handlers/commands/report.js

```diff
@@ -60,6 +60,18 @@ const reportHandler = async ctx => {
 				} ] ] } },
 		);
 	}
+	if (chats.adminLog) {
+		await ctx.telegram.forwardMessage(chats.adminLog, ctx.message.chat.id, ctx.message.reply_to_message.message_id)
+		await ctx.telegram.sendMessage(
```

Comment:
Isn't this a duplicate of the `chats.report` feature?

Reply:
Honestly, it kinda is, but I didn't come up with it.

I did wonder if we should just merge the two, but maybe someone would not care about join messages (leave messages are also supposed to be there but that seems broken, btw), and someone else might care about joins but not admin actions?

Reply:
If you want reports and adminLog in the same chat, you could just enable the two with the same chatId?

Reply:
I think what you should do instead is move the `forwardMessage` logic to `chats.report`. `adminLog` should only be for admin actions.